src/components/ExtraPages/Help/PlotHelp.js

```diff
@@ -1,39 +1,357 @@
+                  reference climatology, which measures the similarity between
+                  the spatial patterns in the maps of simulated and reference
+                  climate; and the standard deviation of the simulated
+                  climatology, which measures the amplitude of the pattern.
+                  These two statistics are plotted on radial axes, with the
+                  correlation coefficient related to the azimuthal angle, and
+                  the standard deviation proportional to the radial distance
+                  from the origin. Where only positive correlations are
+                  observed, the plot will cover a quarter circle; where one or
+                  more runs has a negative correlation with the reference
+                  climatology, the plot will be semicircular.
+                </p>
+                <p>
+                  The standard deviation of the reference climatology is marked
+                  with a heavy dashed arc: points lying close to this arc
+                  represent runs with climatologies that have a similar
+                  amplitude to the reference climatology, while points lying
+                  closer to the origin represent runs that underestimate the
+                  spatial variability, and points lying outside the arc
+                  represent runs that overestimate the spatial variability.
+                  Points close to the x-axis represent runs with high
+                  correlation coefficients, indicating that those runs produce
+                  similar spatial patterns to those in the reference
+                  climatology; points that lie closer to the $y$-axis represent
+                  runs with correlation coefficients close to zero, indicating
+                  very different spatial patterns to those in the reference
+                  field.
+                </p>
+                <p>
+                  Runs represented by points closer to the black star more
+                  closely replicate the the reference climatology in terms of
+                  both the location of spatial features and the range of values
+                  simulated across the UK. During the evaluation period, this is
+                  a desirable trait, indicating that the runs produce realistic
+                  spatial climatologies. For future periods, this indicates that
+                  the spatial patterns are very similar to those produced during
+                  the baseline period. The radial axes are scaled in such a way
+                  that the distance from each point to the black star is
+                  proportional to the centred root mean squared error in the
+                  simulated field; however, note that this centred root mean
+                  squared error excludes UK-wide biases in the runs, which must
+                  be assessed separately using boxplots.
+                </p>
```

Comment:
How about moving all this to a row below? OTherwise there's a large chunk of white space under the plot .

Reply:
sure, I'll do that locally and push